lib/process-services-cloud/src/lib/process/process-filters/components/edit-process-filter-cloud.component.html

```diff
@@ -7,7 +7,7 @@
                     <span *ngIf="showTitle"> {{ 'ADF_CLOUD_EDIT_PROCESS_FILTER.TITLE' | translate}}</span>
                     <div *ngIf="showFilterActions" class="adf-cloud-edit-process-filter-actions">
                         <ng-container *ngIf="toggleFilterActions">
-                            <button *ngFor="let filterAction of processFilterActions" mat-icon-button matTooltip="{{ filterAction.tooltip | translate}}" [attr.data-automation-id]="'adf-filter-action-' + filterAction.actionType" [disabled]="isDisabledAction(filterAction)" (click)="executeFilterActions(filterAction)">
+                            <button *ngFor="let filterAction of processFilterActions" mat-icon-button matTooltip="{{ filterAction.tooltip | translate}}" [attr.data-automation-id]="'adf-filter-action-' + filterAction.actionType" [disabled]="isDisabledAction(filterAction)" (click)="executeFilterActions(filterAction);$event.stopPropagation();">
```

Comment:
can we do this in the component instead of the HTML, passing the $event to the method and stopping propagation there, probably needs unit test as well for that

Reply:
Sure, I added this just for testing the behaviour and forgot to move it to the component. Nice spot!!